crates/map/src/io.rs

```diff
@@ -161,17 +161,17 @@ mod test {
                 )),
             ));
         }
-
+        let path = Path::new("./de_map_test.tar");
         let tmp_dir = Builder::new().prefix("de_map_").tempdir().unwrap();
         let mut tmp_dir_path = PathBuf::from(tmp_dir.path());
         tmp_dir_path.push("test-map.tar");
 
-        task::block_on(store_map(&map, tmp_dir_path.as_path())).unwrap();
-        let loaded_map = task::block_on(load_map(tmp_dir_path.as_path())).unwrap();
+        task::block_on(store_map(&map, &path)).unwrap();
+        let loaded_map = task::block_on(load_map(&path)).unwrap();
```

Comment:
that was only done to create a fake alternative map ; those changes don't belong there.

Reply:
I've done the same trick. Another PR which extends the `de_tools` crate with some kind of simple "generate map" functionality would be appreciated.